opm/autodiff/BlackoilDetails.hpp

```diff
@@ -276,7 +275,7 @@ namespace detail {
                 for ( int idx = 0; idx < np; ++idx )
                 {
                     B_avg[idx] = B.col(idx).sum()/nc;
-                    maxCoeff[idx] = tempV.col(idx).maxCoeff();
+                    //maxCoeff[idx] = tempV.col(idx).maxCoeff();
```

Comment:
valgrind complains about uninitialized values in maxCoeff(). I confess that I've got not much of an idea what needs to be done here to fix it properly, but even with the proposed solution, the Norne results seem to be the same...